Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:buster as base
+
+ENV PYTHONPATH=${PYTHONPATH}:${PWD} 
+RUN pip3 install poetry gunicorn markupsafe==2.0.1
+RUN mkdir /todo_app 
+COPY /todo_app /todo_app
+COPY pyproject.toml /
+COPY .env /
+COPY gunicorn_config.py /
+COPY run.sh /
+COPY poetry.lock /
+WORKDIR /todo_app
+
+RUN poetry config virtualenvs.create false
+RUN poetry install --no-dev
+WORKDIR /
+#RUN chmod +x ./run.sh
+
+FROM base as production
+ENTRYPOINT ["poetry", "run", "gunicorn", "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+EXPOSE 5000
+
+FROM base as development
+ENTRYPOINT [ "poetry", "run", "flask", "run", "--host", "0.0.0.0"]
```

Minor, but given that you run `poetry install --no-dev` above, the development image won't have the dev dependencies as is. Right now there aren't any so it's certainly not an issue, but if a later user adds some this behaviour might well confuse them, so adding an additional install step to catch any dev dependencies for just this target would be a sensible step:
```suggestion
FROM base as development
RUN poetry install
ENTRYPOINT [ "poetry", "run", "flask", "run", "--host", "0.0.0.0"]
```